antifeatures.yml

```diff
@@ -0,0 +1,79 @@
+- id: tracking
+  icon: user-secret
+  title:
+      en: "Tracking"
+      fr: "Pistage"
+  description:
+      en: "Track you and/or report your activity to somewhere, either without your permission or by default."
+
+- id: non-free-network
+  icon: network-wired
+  title:
+      en: "Non-free Network Services"
+      fr : "Services réseau non-libres"
```

Comment:
```suggestion
      en: "Promotes or relies on non-free network services."
      fr : "Promeut ou utilise des services réseau non libres."
```

Reply:
This is the `title` so it should be more concise I guess
I'll modify the description